pages/learn/json-schema-examples.md

```diff
@@ -92,129 +177,414 @@ title: JSON Schema examples
 }
 ```
 
-## Card
+**Data**
+
 ```json
 {
-  "$id": "https://example.com/card.schema.json",
+  "startDate": "2023-08-25T10:00:00Z",
+  "endDate": "2023-08-25T12:00:00Z",
+  "summary": "Conference Presentation",
+  "location": "Conference Center",
+  "recurrenceDule": "FREQ=DAILY;COUNT=5"
+}
+```
+
+
+## Device type
+
+This schema represents electronic devices with a `deviceType` property that determines the device's category, such as `smartphone` or `laptop`. It employs the `oneOf` keyword to dynamically reference schemas based on the `deviceType` property. This flexible schema structure allows data to conform to the appropriate device schema based on the deviceType specified, making it easy to describe different electronic devices with their unique characteristics. When `deviceType` is set to `smartphone`, the schema enforces properties specific to smartphones, and when `deviceType` is set to `laptop`, it enforces properties specific to laptops. 
+
+```json
+{
+  "$id": "https://example.com/device.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "deviceType": {
+      "type": "string"
+    }
+  },
+  "required": ["deviceType"],
+  "oneOf": [
+    {
+      "properties": {
+        "deviceType": { "const": "smartphone" }
+      },
+      "$ref": "https://example.com/smartphone.schema.json"
+    },
+    {
+      "properties": {
+        "deviceType": { "const": "laptop" }
+      },
+      "$ref": "https://example.com/laptop.schema.json"
+    }
+  ]
+}
+
+{
+  "$id": "https://example.com/smartphone.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "brand": {
+      "type": "string"
+    },
+    "model": {
+      "type": "string"
+    },
+    "screenSize": {
+      "type": "number"
+    }
+  },
+  "required": ["brand", "model", "screenSize"]
+}
+
+{
+  "$id": "https://example.com/laptop.schema.json",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "description": "A representation of a person, company, organization, or place",
   "type": "object",
-  "required": [ "familyName", "givenName" ],
   "properties": {
-    "fn": {
-      "description": "Formatted Name",
+    "brand": {
+      "type": "string"
+    },
+    "model": {
+      "type": "string"
+    },
+    "processor": {
       "type": "string"
     },
-    "familyName": {
+    "ramSize": {
+      "type": "number"
+    }
+  },
+  "required": ["brand", "model", "processor", "ramSize"]
+}
+```
+
+**Data**
+
+```json
+{
+  "deviceType": "Smartphone",
```

Comment:
The casing here needs to match what the schema expects.  But the example looks a lot better now.

Reply:
Oh yes, I'll just update that 😅